packages/eslint-plugin/configs/recommended.js

```diff
@@ -13,12 +13,33 @@ const { config: localPrettierConfig } =
 const prettierConfig = { ...defaultPrettierConfig, ...localPrettierConfig };
 
 module.exports = {
+	settings: {
+		'import/resolver': {
+			node: {
+				extensions: [ '.js', '.jsx', '.ts', '.tsx' ],
+			},
+		},
+	},
+	parser: '@typescript-eslint/parser',
```

Comment:
Should it be limited to TS files? Like in:

https://github.com/benmosher/eslint-plugin-import#importparsers

Reply:
I think we can just use a single parser right? `@typescript-eslint/parser` will parse regular JS files just fine, but it's up to you if you prefer to scope it that's fine too.

Reply:
I don't know. I hoped you would know better based on the experience in Calypso 😃

Reply:
Unfortunately I do not. Maybe @jsnajdr or @sirreal or @scinos would have some idea.

We might as well do it if there's an option for it 🤷🏼‍♀️

Reply:
🤷 I don't know, we can try it. I don't think we have anything to use by letting this parser handle everything. I don't know whether there are any downsides.

Reply:
Using `import/parsers` didn't seem to work anyway. Reverting that change back to using a single parser for everything.